src/guarana/schemas/customer.py

```diff
@@ -52,11 +50,18 @@ class CustomerCall(BaseModel):
     notes: Optional[str]
     id_delivery_category: Optional[UUID]
 
-    def __init__(self, **kwargs):
-        kwargs["firstname"] = kwargs["name"]
-        kwargs["address"] = f'{kwargs["street"]} {kwargs["number"]} {kwargs["complement"]}'
-        kwargs["bairro"] = kwargs["neighborhood"]
-        kwargs["phone"] = kwargs["cellphone"]
-        kwargs["zip"] = kwargs["cep"]
+    @root_validator
+    def build_address(cls, values) -> dict:
+        address_string = ""
+        if values["street"]:
+            address_string = values["street"]
+        else:
+            return values
+        if values["number"]:
+            address_string += f", {values['number']}"
+        if values["complement"]:
+            address_string += f" - {values['complement']}"
+
+        values["address"] = address_string
```

Comment:
[Hacky McHackington](https://daniellenz.blog/2021/02/20/computed-fields-in-pydantic/)